src/views/misc/Terms.vue

```diff
@@ -0,0 +1,127 @@
+
+    <h2>5. Prohibitions</h2>
+
+<p>You may not publicly represent or imply that either Phoenix or MorphoBank is participating in, or has sponsored, approved, or endorsed the manner or purpose of your use of the Data. You may not use any trademark or logo of Phoenix or MorphoBank, including any such trademarks or logos contained on or constituting a part of MorphoBank, without prior written permission from Phoenix.</p>
+
+<p>Phoenix does not endorse or recommend any commercial products, processes, or services. The views and opinions of authors expressed on MorphoBank may not be used for advertising or product endorsement purposes. Some web pages contained within MorphoBank may provide links to other Internet sites only for the convenience of our users. Phoenix and MorphoBank are not responsible for the availability or content of these external sites, nor do Phoenix and MorphoBank endorse, warrant or guarantee the content, products, services or information described or offered at these third-party Internet sites.</p>
+
+<p>You may not use MorphoBank or the Services in any unlawful manner or in any manner that would impair the provision of the Services to others or that would compromise the security or functionality of MorphoBank or the Services. You may not post or submit any content or Data that is offensive, defamatory, illegal, or intentionally misleading or fraudulent. You may not post or submit any content or Data which comprises advertisement or solicitation.</p>
+
+<p>You shall not (and shall not permit any third party to) either:</p>
+<ol>
+  <li>Take any action or</li>
+  <li>Upload, download, post, submit or otherwise distribute or facilitate distribution of any Data or other content on or through MorphoBank or the Services that:</li>
+</ol>
+<ul>
+  <li>Infringes any patent, trademark, trade secret, copyright, right of publicity or other right of any other person or entity or violates any law or contractual duty;</li>
+  <li>You know is false, misleading, untruthful or inaccurate;</li>
+  <li>Is unlawful, threatening, abusive, harassing, defamatory, libelous, deceptive, fraudulent, invasive of another's privacy, tortious, obscene, offensive, or profane;</li>
+  <li>Contains software viruses or any other computer codes, files, or programs that are designed or intended to disrupt, damage, limit or interfere with the proper function of any software, hardware, or telecommunications equipment or to damage or obtain unauthorized access to any system, data, password or other information of Phoenix Bioinformatics, any other users of MorphoBank, or any third party;</li>
+  <li>Impersonates any person or entity, including any employee or representative of Phoenix Bioinformatics.</li>
+</ul>
+<p>Additionally, you shall not:</p>
+<ol type="A">
```

Comment:
It seems the ol and li does not line up very well. Each "ABCD" has a weird 0 following them

Reply:
Also, can you make the ol and li styles consistent across the different sections? Some use latter which some use number; Some are split into li and some are not (see my comment below)